src/Viewer/components/Monaco/MonacoInstance.js

```diff
@@ -100,21 +98,17 @@ function MonacoInstance ({
 }) {
     const {theme} = useContext(ThemeContext);
     const editorRef = useRef(null);
-    const monacoRef = useRef(null);
     const timeoutRef = useRef(null);
-    const [monacoTheme, setMonacoTheme] = useState("customLogLanguageLight");
-
-    const [language, setLanguage] = useState("");
-
-    loader.config({monaco});
+    const editorContainerRef = useRef(null);
```

Comment:
Might make slightly more sense to move this editorContainerRef declaration directly below editorRef